vendor/k8s.io/klog/v2/internal/serialize/keyvalues.go

```diff
@@ -44,53 +46,49 @@ func WithValues(oldKV, newKV []interface{}) []interface{} {
 	return kv
 }
 
-// TrimDuplicates deduplicates elements provided in multiple key/value tuple
-// slices, whilst maintaining the distinction between where the items are
-// contained.
-func TrimDuplicates(kvLists ...[]interface{}) [][]interface{} {
```

Comment:
did this method go away? how did that pass the klog apidiff compatibility checks?

Reply:
edit: oh... it's under internal/..., nevermind

Reply:
I had to get apidiff fixed first to have it accept this change without complaining, but yes, this is exactly the reason :smile: